src/test/groovy/com/github/vkorobkov/jfixturescmd/utils/PrintStreamWrapper.groovy

```diff
@@ -0,0 +1,19 @@
+package com.github.vkorobkov.jfixturescmd.utils
+
+class PrintStreamWrapper extends PrintStream {
+    private StringBuilder content = new StringBuilder()
+
+    PrintStreamWrapper(OutputStream out) {
+        super(out)
```

Comment:
Can we use `java.io.ByteArrayOutputStream` to pass to the super class and get the content from it?